src/main/java/com/minecolonies/coremod/entity/ai/util/RecipeStorage.java

```diff
@@ -65,7 +66,7 @@ public RecipeStorage(final List<ItemStack> input, final int gridSize, final Item
      */
     public List<ItemStack> getSecondaryOutput()
```

Comment:
If you want to make a copy of the outputs, thats probably okay. I can think of times when we won't need copies, but we probably will most of the time. I'm including the primary output when I say that as well.